pyquaticus/envs/pyquaticus.py

```diff
@@ -1734,29 +1721,60 @@ def _render(self):
                 )
 
             for player in teams_players:
+                # add pos to player_poses
+                player_poses[color].append(player.pos)
+
+                # render tagging
+                player.render_tagging(self.tagging_cooldown)
+
+                # heading
                 orientation = Vector2(list(mag_heading_to_vec(1.0, player.heading)))
                 ref_angle = -orientation.angle_to(self.UP)
 
                 # transform position to pygame coordinates
-                pos_x_m, pos_y_m = player.pos
-                pos_x, pos_y = pos_x_m * self.pixel_size, pos_y_m * self.pixel_size
-                pos_x += self.arena_offset
-                pos_y = (
-                    0.5 * self.arena_height
-                    - (pos_y - 0.5 * self.arena_height)
-                    + self.arena_offset
-                )
-                blit_position = np.array([pos_x, pos_y])
+                blit_position = self.world_to_screen(player.pos)
                 rotated_surface = rotozoom(player.pygame_agent, ref_angle, 1.0)
                 rotated_surface_size = np.array(rotated_surface.get_size())
                 blit_position -= 0.5 * rotated_surface_size
                 self.screen.blit(rotated_surface, blit_position)
 
+        # visually indicate distances between players of both teams 
+        assert len(player_poses) == 2, "If number of teams > 2, update code that draws distance indicator lines"
```

Comment:
Do you think it would make sense to only draw the line to the closest agent? Or would that be confusing if it switches often?

Reply:
I was trying to match the Moose visualization (where I think it is drawing the lines to all agents within a certain radius). It might get confusing if it switches back and forth a lot, and I think if we leave it as it is it won't look too cluttered for 2v2. But if it still does than I can decrease the distance threshold at which the code begins to draw lines between agents.

Reply:
Got it! It looks good how it is. I realized I misunderstood the assert. I thought it was about the number of players on the team but it’s actually about the number of teams, which we can safely assume is 2 (at least for now). Thanks!